loky/reusable_executor.py

```diff
@@ -73,6 +73,17 @@ def get_reusable_executor(max_workers=None, context=None, timeout=10,
     with _executor_lock:
         global _executor, _executor_args
         executor = _executor
+
+        if max_workers is None:
+            if reuse is True and executor is not None:
+                max_workers = executor._max_workers
+            else:
+                max_workers = cpu_count()
```

Comment:
This behavior need to be documented. Maybe in another PR, with the `initializer` and affinity.

Reply:
+1 :)